FEM/fem_ele_std.cpp

```diff
@@ -9554,6 +9554,12 @@ void CFiniteElementStd::CalcSaturation(MeshLib::CElem& elem)
 	MediaProp = mmp_vector[mmp_index];
 	MediaProp->m_pcs = pcs;
 	MediaProp->Fem_Ele_Std = this;
+	// For variable dependent fluid models, which needs variable interpolation.
+	for (std::size_t i=0; i < mfp_vector.size(); i++)
```

Comment:
why not use `CFiniteElementStd::SetMaterial(int phase)`?

Reply:
Thanks. Changed accordingly.